index.js

```diff
@@ -1,30 +1,35 @@
-import 'dotenv/config';
-import {dbInput, newBarInput} from './Inputs/config.js';
-import { loadPositions } from "./DBConnection/loadPositions.js"
-import {initDB, initClient} from './DBConnection/initPool.js';
-import {loadSettlementPrice, loadMarkPrice} from './DBConnection/loadPrice.js';
-import { loadOrders } from './DBConnection/loadOrders.js';
-import { barInit } from './utilities/isNewBar.js';
-import logger from './logger.js';
-import { checkOpenOrder } from './DBConnection/checkOpenOrder.js';
+import "dotenv/config";
+import { dbInput, newBarInput, fetchActiveSymbol, symbols, symbolsForMarkPrice } from "./Inputs/config.js";
```

Comment:
remove unnecessarily imports
Btw, highly recommend to use something like ESlint to help you maintain your code organized.